frontend/webpack.circular-deps.ts

```diff
@@ -90,7 +101,10 @@ const applyThresholds = (
   }
 
   const minLengthCycles = minLengthCycleCount(cycles);
-  if (thresholds.minLengthCycles && minLengthCycles > thresholds.minLengthCycles) {
+  if (
+    typeof thresholds.minLengthCycles === 'number' &&
+    minLengthCycles > thresholds.minLengthCycles
```

Comment:
Would this be simpler as

```suggestion
    !thresholds.minLengthCycles ||
    minLengthCycles > thresholds.minLengthCycles
```

Reply:
Hm, maybe this is wrong if `minLengthCycles` is `0`

Reply:
Unfortunately I think tha twouldn't work since `thresholds.minLengthCycles` is set to 0, so it `!0` would be true

Reply:
> Unfortunately I think that wouldn't work since thresholds.minLengthCycles is set to 0, so !0 would be true and no error would be emitted

In that case the error is always emitted, yes?

I think the problem is if `minLengthCycles` is `0`, the error is still emitted.